planning/static_centerline_generator/src/centerline_source/optimization_trajectory_based_centerline.cpp

```diff
@@ -116,7 +116,7 @@ OptimizationTrajectoryBasedCenterline::generate_centerline_with_optimization(
 }
 
 std::vector<TrajectoryPoint> OptimizationTrajectoryBasedCenterline::optimize_trajectory(
-  const Path & raw_path) const
```

Comment:
Any reason to remove const here?

Reply:
Same as https://github.com/autowarefoundation/autoware.universe/pull/6817#discussion_r1567585660 do you want to revert the change despite the `clang-tidy` warning?

Reply:
If there is no reason except for just clang-tidy, yes.